jobs/clock.js

```diff
@@ -0,0 +1,10 @@
+const cron = require('node-cron');
+const emailChallengeList = require('./email-challenge-list.js');
+
+// Every 5 minutes, from 8:00AM - 7:00 PM ETC from Monday through Friday
```

Comment:
I've been going back and fourth on how long the job should run. For full flexibility, it probably should just be between 8:00 AM - 7:00PM everyday